app/Http/Controllers/API/v1/HealthMonitorController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers\API\v1;
+
+use App\Enum\CacheKey;
+use App\Http\Controllers\HafasController;
+use Illuminate\Database\QueryException;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Queue;
+
+class HealthMonitorController
+{
+
+    public function healthReady(): JsonResponse {
+        $checks = [
+            "database_reachable" => $this->isDatabaseReachable()
+        ];
+
+        return response()
+            ->json($checks)
+            ->setStatusCode(array_product($checks) ? 200 : 500);
+    }
+
+    public function healthFitness(): JsonResponse {
+        $checks = [
+            "database_reachable" => $this->isDatabaseReachable(),
+            "queue_running"      => $this->isQueueRunning(),
+            "db_rest_reachable"  => $this->isDbRestReachable(),
+            "scheduler_running"  => $this->isSchedulerRunning(),
+        ];
+
+        return response()
+            ->json($checks)
+            ->setStatusCode(array_product($checks) ? 200 : 500);
+    }
+
+    private function isDatabaseReachable(): bool {
+        try {
+            DB::connection()->getPdo();
+            return true;
+        } catch (QueryException $e) {
+            Log::info("Health Monitor ran into Query exception. Is the database not available?");
+            return false;
+        }
+    }
+
+    private function isQueueRunning(): bool {
+        $size = Queue::size();
+        return $size < 10;
```

Comment:
This should be increased in my opinion, since our brouter tasks take a few seconds to retrieve. If many users check in parallel, the number increases quickly.

Reply:
I think, I'd like to be conservative and keep the number quite low. When it starts to bug us, we can still update the check or set-up a second queue worker (the jobs are independent from each other and it should be easy to duplicate the systemd service, even on the old hardware).